idunn/utils/default_settings.yaml

```diff
@@ -91,6 +91,7 @@ CORS_OPTIONS_REQUESTS_ENABLED: False
 
 #######################
 ## Directions
+DIRECTIONS_ALLOWED_CITIES: ['paris', 'lyon']
```

Comment:
So we limit this to only a few cities in the end? I thought it was only for public transports though. Did I miss something?

Reply:
No you're right, my settings doesn't have a correct name!